kml/serdes_binary.hpp

```diff
@@ -251,7 +251,7 @@ class DeserializerKml
     NonOwningReaderSource source(reader);
     m_header.Deserialize(source);
 
-    if (m_header.m_version == Version::V8)
+    if (m_header.m_version == Version::V8 || m_header.m_version == Version::V9)
```

Comment:
```suggestion
    // The recent MapsMe update increased the version number, but it is not clear yet what changed/added in a newer version.
    // Revise V9 in case of discovered crashes.
    if (m_header.m_version == Version::V8 || m_header.m_version == Version::V9)
```